client.js

```diff
@@ -224,9 +261,20 @@ class Client {
    *   { dataConditions: { "Text": "Invalid or missing feature license detected." } }
```

Comment:
Should document here all fields that can be passed into the query. I see you wrote something to `_validateEventQuery` but end users probably will read just `requestEvents` doc not the internal methods doc.

Reply:
Also, I think the example should include how to set multiple dataConditions.

BTW, I discovered a server-side error when testing multiple dataConditions - I'll try to look at it soon. So you may have some trouble testing this feature at the moment.